Package.swift

```diff
@@ -149,7 +149,7 @@ let package = Package(
       url: "https://github.com/google/GoogleAppMeasurement.git",
       // Note that CI changes the version to the head of main for CI.
       // See scripts/setup_spm_tests.sh.
-      .exact("8.12.0")
+      .branch("main")
```

Comment:
Shouldn't be checked in. We really need to do https://github.com/firebase/firebase-ios-sdk/issues/9072

Reply:
Thanks. Done in 0826e4b42c1d929a4b926180287d428ecb8f7a58.